docs/index.md

```diff
@@ -1,3 +1,27 @@
-template: ecosystem.html
+Welcome to `Ansible ecosystem documentation`.
 
-# Ansible ecosystem
+Here you can find `How To` guides and `tutorials` for cross-project use cases within the Ansible ecosystem in this docsite.
+
+If you're looking for documentation related to a specific Ansible project, visit the [ecosystem page](ecosystem.md).
+You can explore Ansible projects and find links to the related documentation.
+
+## Extending Ansible automation
+
+Ansible is a programming language and runtime plus an entire ecosystem of projects that extending automation capabilities to a virtually unlimited set of use cases.
```

Comment:
```suggestion
Ansible is a programming language and runtime plus an entire ecosystem of projects that extends automation capabilities to a virtually unlimited set of use cases.
```

Reply:
I'm unsure about the rest of the wording on this one. I don't know if we call Ansible a programming language and runtime in other places?  Also, I'd change 'virtually unlimited' to something less grand like 'an extensive and extendable" set of use cases.